test/router/modules/pool/LinkedPoolVelodromeV2ModuleOptUSDC.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {LinkedPoolIntegrationTest} from "./LinkedPoolIntegration.t.sol";
+
+import {VelodromeV2Module} from "../../../../contracts/router/modules/pool/velodrome/VelodromeV2Module.sol";
+
+contract LinkedPoolVelodromeV2ModuleArbUSDCTestFork is LinkedPoolIntegrationTest {
+    string private constant OPT_ENV_RPC = "OPTIMISM_API";
+    // 2023-07-11
+    uint256 public constant OPT_BLOCK_NUMBER = 106753037;
+
+    // Velodrome V2 USDC/USDT pool on Optimism
+    address public constant VEL_V2_USDCUSDT_POOL = 0x2B47C794c3789f499D8A54Ec12f949EeCCE8bA16;
+
+    // Velodrome V2 Router on Optimism
+    address public constant VEL_V2_ROUTER = 0xa062aE8A9c5e11aaA026fc2670B0D65cCc8B2858;
+
+    // nUSD/USDC DefaultPool on Optimism
+    address public constant NUSD_POOL = 0xF44938b0125A6662f9536281aD2CD6c499F22004;
+
+    // Native USDC on Optimism
+    address public constant USDC = 0x7F5c764cBc14f9669B88837ca1490cCa17c31607;
+    // nUSD on Optimism
+    address public constant NUSD = 0x67C10C397dD0Ba417329543c1a40eb48AAa7cd00;
+    // Native USDT on Optimism
+    address public constant USDT = 0x94b008aA00579c1307B0EF2c499aD98a8ce58e58;
+
+    VelodromeV2Module public velodromeV2Module;
+
+    constructor() LinkedPoolIntegrationTest(OPT_ENV_RPC, OPT_BLOCK_NUMBER) {}
+
+    function afterBlockchainForked() public override {
+        velodromeV2Module = new VelodromeV2Module(VEL_V2_ROUTER);
+    }
+
+    function addExpectedTokens() public override {
+        // Expected order of tokens:
+        // 0: USDC
+        // 1: USDT
+        // 2: nUSD
+        addExpectedToken(USDC, "USDC");
+        addExpectedToken(USDT, "USDT");
+        addExpectedToken(NUSD, "nUSD");
+    }
+
+    function addPools() public override {
+        addPool({
+            poolName: "USDC/USDT",
+            nodeIndex: 0,
+            pool: VEL_V2_USDCUSDT_POOL,
+            poolModule: address(velodromeV2Module)
+        });
+        addPool({poolName: "nUSD/USDC", nodeIndex: 0, pool: NUSD_POOL}); // Q: why is this node index 0?
```

Comment:
@ChiTimesChi question for you here as I'm not understanding exactly how node index specification works for this function

Reply:
In this scenario you're attaching two pools to `USDC` node (which is the root token in example):
- VelodromeV2 USDC/USDT (attached to USDC node with index 0)
- DefaultPool nUSD/USDC (attached to USDC node with index 0)

Which results in the following structure:
![image](https://github.com/synapsecns/synapse-contracts/assets/88190723/416ef256-e6df-49d7-b3a1-89c937011180)

In production, however, the root token would be nUSD, and the setup would be different:
- DefaultPool nUSD/USDC (attached to nUSD node with index 0)
- VelodromeV2 USDC/USDT (attached to USDC node with index 1)

Which would result in the following structure:
![image](https://github.com/synapsecns/synapse-contracts/assets/88190723/c9e80d5d-949b-4c8e-8f4a-ca04709ee1ab)

Reply:
As mentioned in #245, the ideal test here would be one that mimics the upcoming production as much as possible, so changing the expected tokens order and order of pools addition should be good practice.